pkg/nvcdi/spec/builder.go

```diff
@@ -55,7 +56,7 @@ func newBuilder(opts ...Option) *builder {
 	}
 
 	if s.version == "" {
-		s.version = DetectMinimumVersion
+		s.version = string(DetectMinimumVersion)
```

Comment:
Wouldn't it be better to a a string method to `minimum` type?

Reply:
I don't know whether it would be better. The implementation would be:
```
func (v minimum) String() string {
    return string(v)
}
```
which would cast anyway.

Reply:
"better" was a wrong word, yeah same functionality. I was just wondering if it would make sense to have it that way, but it's ok